src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.Patch.cs

```diff
@@ -192,19 +267,12 @@ private bool TryMergePatches(string id, PatchRequest patchRequest)
                 newVals[kvp.Key] = kvp.Value;
             }
 
-            Defer(new PatchCommandData(id, null, new PatchRequest
-            {
-                Script = newScript,
-                Values = newVals
-            }, null));
+            Defer(new PatchCommandData(id, null, new PatchRequest {Script = newScript, Values = newVals}, null));
 
```

Comment:
In general, you need to fix the IDE settings on those to match our coding conventions

Reply:
👌